content/community.json

```diff
@@ -71,20 +71,12 @@
   ],
   "events": [
     {
-      "url": "https://dssconf.pl/",
```

Comment:
I think, no need to remove anything. Just add a new event.

Reply:
Gotcha!  Put it back.